test/test_torch.py

```diff
@@ -17374,6 +17380,8 @@ def test_split_view(self, device):
 
 _float_types_no_half = [torch.float, torch.double]
 
+_complex_types = [torch.cfloat, torch.cdouble]
```

Comment:
ha! well I added _complex types here https://github.com/pytorch/pytorch/pull/38400/files#diff-9996665f82f52030836eb8657057cfadR17295 but I don't wanna block this PR because of this. let's remove it in the subsequent PR